actions/tag/routes.go

```diff
@@ -0,0 +1,22 @@
+package tag
+
+import (
+	"github.com/go-chi/chi"
+)
+
+type TagRouter struct {}
+
+
+func (tag TagRouter) Router() chi.Router {
```

Comment:
Why TagRouter receiver?

Reply:
Instead of having receiver function. we can have have function like *TagRouter*

```
func TagRouter() http.Handler {
	r := chi.NewRouter()
	r.Get("/", getTags)    // GET /tag - return list of tags
	r.Post("/", createTag) // POST /tag - create a new tag and persist it

	r.Route("/{id}", func(r chi.Router) {
		r.Get("/", getTagByID)   // GET /tags/{id} - read a single tag by :id
		r.Delete("/", deleteTag) // DELETE /tags/{id} - delete a single tag by :id
	})

        return r
}
```